README.md

```diff
@@ -62,7 +62,29 @@ Check out [contribution guidelines](CONTRIBUTING.md) and other [awesome](https:/
 - [2007] [Innovation in Indie Games](https://www.youtube.com/watch?v=PR-ZtrOGHiY): Kyle Gabler, Jenova Chen, Jon Mak, Jon Blow (Panel)
 - [2000] [It's Great to be Back! Fast Code, Game Programming, and Other Thoughts from 20 (Minus Two) Years in the Trenches](http://www.gdcvault.com/play/1016641/): Michael Abrash (Rad Game Tools)
 
+### TED Talks
+
+[http://www.ted.com/topics/gaming](http://www.ted.com/topics/gaming)
+
+- [2015, TED]  [Physical therapy is boring — play a game instead](http://www.ted.com/talks/cosmin_mihaiu_physical_therapy_is_boring_play_a_game_instead): Cosmin Mihaiu (Indie) 
+- [2015, TEDxTokyo] [Play this game to come up with original ideas](http://www.ted.com/talks/shimpei_takahashi_play_this_game_to_come_up_with_original_ideas): Shimpei Takahashi 
+- [2013, TEDGlobal] [Massively multi-player… thumb-wrestling?](http://www.ted.com/talks/jane_mcgonigal_massively_multi_player_thumb_wrestling): Jane McGonigal (Institute for the Future) 
+- [2012, TEDGlobal] [The Game That Can Give You 10 Extra Years Of Life](https://www.ted.com/talks/jane_mcgonigal_the_game_that_can_give_you_10_extra_years_of_life?language=en): Jane McGonigal (Indie) 
```

Comment:
I did the parens for if its a company or school so I think it's fine to just drop the (Indie) since no () I guess would imply there not associated with anything which can represent indie.